.github/workflows/python-bindings.yml

```diff
@@ -46,7 +46,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        runs-on: [ubuntu-latest, macos-13, macos-14, windows-latest]
+        runs-on: [ubuntu-latest, macos-13, macos-14, macos-15, windows-latest]
```

Comment:
Why use `macos-15` here but `macos-15-xlarge` above?

Reply:
We only need one Intel and one ARM macOS runner here

Reply:
> We only need one Intel and one ARM macOS runner here

so if I choose ``macos-15-xlarge``, it is okay, right? 

macos-15-xlarge: arm
macos-14: x64

Reply:
In that case, `macos-13` is superfluous. Again, we only need one Intel and one ARM runner here for macOS.

Reply:
ah, I see! Now, I know what you mean. Thanks!

Reply:
I just double-checked with GitHub's documentation. `macos-13` is the only tag that has Intel runners available. `macos-14` and `macos-15` are M1-based, i.e., ARM. The `xlarge` runners we cannot use because they are only available for Enterprise customers.

Reply:
Oh, thank you for double-checking.